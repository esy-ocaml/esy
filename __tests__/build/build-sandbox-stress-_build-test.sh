doTest () {
  initFixture sandbox-stress-_build
  run esy build
  run esy x echo ok
}
