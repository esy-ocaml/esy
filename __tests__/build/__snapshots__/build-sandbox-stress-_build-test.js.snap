// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`build sandbox-stress 1`] = `
"RUNNING: esy config ls
esy:bin no .esyrc found
esy:bin prefix: <testRoot>/esy
RUNNING: esy build
esy:bin no .esyrc found
esy:bin prefix: <testRoot>/esy
esy:bin needRebuildTarget <sandbox>/node_modules/.cache/_esy/build/bin/build
esy:bin:esyBuild execute
esy:bin no .esyrc found
esy:bin prefix: <testRoot>/esy
esy:build-dependencies checking if dependencies are built
RUNNING: esy x echo ok
esy:bin no .esyrc found
esy:bin prefix: <testRoot>/esy
esy:bin needRebuildTarget <sandbox>/node_modules/.cache/_esy/build/bin/build
esy:bin no .esyrc found
esy:bin prefix: <testRoot>/esy
esy:build-dependencies checking if dependencies are built"
`;
